app/assets/stylesheets/application.css

```diff
@@ -13,3 +13,88 @@
  *= require_tree .
  *= require_self
  */
+
```

Comment:
We can reuse these classes between the merchants web pages and the admin web pages, if you want